api/controllers/game/ready.js

```diff
@@ -103,6 +103,7 @@ module.exports = function (req, res) {
             change: 'ready',
             userId: user.id,
             pNum: user.pNum,
+            gameId: game.id,
```

Comment:
Added this so we can filter out socket events for games we aren't viewing right now